repo/format.go

```diff
@@ -0,0 +1,36 @@
+package repo
+
+import (
+	"fmt"
+	"strings"
+
+	"gopkg.in/src-d/go-git.v4/plumbing/object"
+)
+
+// FormatMessage formats the commit's message
+func FormatMessage(c *object.Commit, firstLinePrefix string, nextLinesPrefix string, lineSeparator string) string {
+	outputStrings := []string{}
+	maxLines := 6
+
+	lines := strings.Split(c.Message, "\n")
+	for _, line := range lines {
+		if strings.TrimSpace(line) == "" {
+			continue
+		}
+
+		prefix := ""
```

Comment:
var prefix string is actually preferred.

Reply:
👍